src/app/modules/group/http-services/create-group-invitations.service.ts

```diff
@@ -2,9 +2,15 @@ import { Injectable } from '@angular/core';
 import { Observable } from 'rxjs';
 import { environment } from 'src/environments/environment';
 import { HttpClient } from '@angular/common/http';
-import { ActionResponse, successData, objectToMap } from 'src/app/shared/http-services/action-response';
+import { ActionResponse, successData } from 'src/app/shared/http-services/action-response';
 import { map } from 'rxjs/operators';
 
+export enum InvitationResult {
+  success,
+  error,
+  already_invited,
+  not_found,
+}
```

Comment:
I was suggesting an union type but in this case probably an enum is better, right 👍 
However, enum options should be in PascalCase, so `Success`,  `AlreadyInvited`, ...